test/unit/utils.test.js

```diff
@@ -344,13 +344,14 @@ describe('basicQuery', () => {
       expect(basicQuery(defaultQueryObj)).to.be.a('function');
     });
 
-    it('should take empty objects as input and return default string when invoked', () => {
+    // Why was this behavior originally implemented?
+    xit('should take empty objects as input and return default string when invoked', () => {
```

Comment:
A long, long time ago, in a department far, far away, the desired behavior was to not allow empty searches but also let power users use `*` to mean "search for all".
* No Empty Searches - https://jira.nypl.org/browse/DIS-153
* Empty Search Trick - https://jira.nypl.org/browse/DIS-154

_I think_ there was a mapping from "*" in the client to "q=" in the server request to allow this.

In the return of the developer, empty searches was the desired behavior. This probably removed the "*" in the client-side but kept the "q=" in the server.

Hopefully, this is enough to describe why the behavior was originally implemented. If it needs to be updated for whatever is best now, go for it. Do document it.

Reply:
Thanks, Edwin. It's helpful for taking accountability on these UX decisions. These options are still functional. We just don't provide a Breadcrumb link to them.

@thisisstephenbetts some relevant past tickets for the Breadcrumb and Search behavior we have discussed.